plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml

```diff
@@ -6,3 +6,6 @@ title: Eclipse Theia for Eclipse Che
 description: Eclipse Theia
 icon: https://raw.githubusercontent.com/theia-ide/theia/master/logo/theia-logo-no-text-black.svg?sanitize=true
 url: https://github.com/ws-skeleton/che-editor-theia/releases/download/untagged-cf91340b0b4af08ec756/che-editor-plugin.tar.gz
+publisher: Red Hat, Inc.
+category: Editor
+repository: https://github.com/ws-skeleton/che-editor-theia/
```

Comment:
Location of the repo is about to change. I suggest changing it right away not to need to update later. See https://github.com/eclipse/che-theia/pull/48

Reply:
changed, thanks